CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ link_directories( DemoGafferExtension ${GAFFER_ROOT}/lib )
 add_library( DemoGafferExtension SHARED ${DemoGafferExtensionSrc} )
 target_compile_definitions( DemoGafferExtension PRIVATE BOOST_SIGNALS_NO_DEPRECATION_WARNING=1 )
 target_include_directories( DemoGafferExtension PRIVATE include ${GAFFER_ROOT}/include )
-target_link_libraries( DemoGafferExtension Gaffer GafferScene )
+target_link_libraries( DemoGafferExtension Gaffer GafferScene IECore IECorePython boost_python Python)
```

Comment:
Adding IECore is fine here, but the rest should be added below in the python module (line 27). We don’t want the base lib linking to python.